Packs/cisco-meraki/Integrations/CiscoMerakiv2/CiscoMerakiv2.yml

```diff
@@ -0,0 +1,2283 @@
+      - deny
+    - name: syslog_enabled
+      description: Whether to log this rule to syslog, only applicable if a syslog has been configured.
+      auto: PREDEFINED
+      predefined:
+      - 'false'
+      - 'true'
+    outputs:
+    - contextPath: CiscoMeraki.L3FirewallRule.networkId
+      description: ID of the network.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.comment
+      description: Comment for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.policy
+      description: Policy for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.protocol
+      description: Protocol for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.destPort
+      description: Destination port for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.destCidr
+      description: Destination CIDR for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.srcPort
+      description: Source port for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.srcCidr
+      description: Source CIDR for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L3FirewallRule.rules.syslogEnabled
+      description: Whether syslog is enabled.
+      type: Boolean
+  - name: meraki-network-l3firewall-rule-delete
+    description: Delete the L3 firewall rules from an MX network.
+    arguments:
+    - name: network_id
+      description: ID of the network to delete all the L3Firewall rules for. Use `meraki-network-list` to fetch all network IDs. This overrides the network ID instance parameter.
+      default: true
+  - name: meraki-network-l7firewall-rule-list
+    description: List the MX L7 firewall rules for an MX network.
+    arguments:
+    - name: network_id
+      description: ID of the network to retrieve from. Use `meraki-network-list` to fetch all network IDs. This overrides the network ID instance parameter.
+      default: true
+    outputs:
+    - contextPath: CiscoMeraki.L7FirewallRule.networkId
+      description: ID of the network.
+      type: String
+    - contextPath: CiscoMeraki.L7FirewallRule.rules.policy
+      description: Policy for the rule.
+      type: String
+    - contextPath: CiscoMeraki.L7FirewallRule.rules.type
+      description: Type of the rule.
+      type: String
+    - contextPath: CiscoMeraki.L7FirewallRule.rules.value
+      description: Value for the chosen type.
+      type: String
+  - name: meraki-network-l7firewall-rule-update
+    description: Update the MX L7 firewall rules for an MX network.
+    arguments:
+    - name: network_id
+      description: ID of the network to update from. Use `meraki-network-list` to fetch all network IDs. This overrides the network ID instance parameter.
+      default: true
+    - name: override
+      description: Whether to override the current rules or append to them.
```

Comment:
I think this should be: Whether to override the current rules. If false, will append to the current rules.